Dockerfile

```
@@ -0,0 +1,16 @@
+# base image for python
+FROM python:3.10-alpine3.16
+
```

```suggestion
WORKDIR /usr/app
```

I think it would also be a good practice to set a working directory for our application.